pkgs/development/python-modules/dogtail/default.nix

```diff
@@ -32,8 +37,9 @@ buildPythonPackage rec {
     ./nix-support.patch
   ];
 
-  nativeBuildInputs = [ gobject-introspection dbus xvfb_run ]; # for setup hooks
-  propagatedBuildInputs = [ at-spi2-core gtk3 pygobject3 pyatspi pycairo ];
+  nativeBuildInputs = [ gobject-introspection dbus xvfb_run wrapGAppsHook ]; # for setup hooks
+  propagatedBuildInputs = [ at-spi2-core gtk3 pygobject3 pyatspi pycairo pango.out librsvg ];
```

Comment:
Do we still need `pango.out` and `librsvg`?

Reply:
not anymore, fixed.